charts/hathor-fullnode/values.yaml

```diff
@@ -0,0 +1,103 @@
+# Default values for hathor-full-node.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+hathor:
+  network: testnet
+  snapshotUrl: https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-latest
+  # For mainnet:
+  # snapshotUrl: https://hathor-public-files.s3.amazonaws.com/temp/mainnet-data-latest
```

Comment:
Can it be a variable where we concatenate the base snapshotURL with the network set in the line above?

Reply:
Yes, I think that is better.